tests/lang/tests/arithmetics_mul.yaiwr

```diff
@@ -1,4 +1,5 @@
 // Run-time:
-//   stdout: 27
+//    stdout: 27
+//    stderr:
```

Comment:
So you are explicitly checking that stderr is empty?

Reply:
yes